src/app/csc/csc.component.html

```diff
@@ -70,79 +65,62 @@ <h4 class="no_data">
         class="csc-projects"
         *ngIf="!dataLoading && filteredCscProjectsList.length > 0"
       >
-        <div class="d-none d-lg-block">
-          <ng2-smart-table
-            class="table table-striped"
-            [settings]="settings"
-            [source]="filteredCscProjectsList"
-          ></ng2-smart-table>
-        </div>
-      </div>
-    </div>
-    <!-- ********************< Mobile Cards*****************************************************-->
-
-    <div
-      class="d-lg-none col-xs-10"
-      *ngIf="!dataLoading && filteredCscProjectsList.length > 0"
-    >
-      <div class="card" *ngFor="let project of filteredCscProjectsList">
-        <!--Card content-->
-        <div class="card-block mobile">
-          <!--Title-->
-          <h4 class="card-title">
-            <a [routerLink]="['/project/', sbId, project.id]">
-              {{ project["title"] }}
-            </a>
-            <ul *ngIf="project.children?.length > 0">
-              <li *ngFor="let childProject of project.children">
-                {{ childProject.title }}
-              </li>
-            </ul>
-          </h4>
-
-          <!--Project Year-->
-          <div class="card-text">
-            <b>Funding Year: </b>
-            <span *ngIf="project['fiscal_year']">
+        <table
```

Comment:
Formatting needs to match production site

Reply:
Done!